.github/workflows/ci.yaml

```
@@ -29,6 +29,40 @@ jobs:
+    timeout-minutes: 8
+    env:
+      UV_PYTHON: ${{ matrix.python-version }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v6.0.2
+        with:
+          persist-credentials: false
+
+      - name: Install tooling
+        uses: ./.github/actions/install_deps
```

actionlint (1.7.11)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/install_deps": line 4: unexpected key "type" for definition of input "skip-updating-devcontainer-hash"

(action)